draft-ietf-ppm-dap.md

```diff
@@ -1006,9 +1006,14 @@ the following parameters associated with it:
   are grouped into batches and the properties that all batches for this task
   must have. The party MUST NOT configure the task if it does not recognize the
   batch mode.
+* `task_start`: The start timestamp of a task. Clients MAY opt out of the task
```

Comment:
semantic nit: `task_start` and `task_expiration` are effectively the start and end of the time interval in which the task is accepting report uploads, but "start" and "expiration" are not opposites.

Maybe `task_start_time` and `task_end_time`? (or `task_start` & `task_end`?)

Reply:
Works for me.

Reply:
Fixed. Made a pass to change `task_expiration` to `task_end`, and `expired` to `ended`.